### PR TITLE
Support PaddleSlim quantitative paddle model export to ONNX format

### PR DESCRIPTION
**Pull request by yeliang2258**
1. Support PaddleSlim quantitative model export to ONNX format
2. Add remove_isolated_node_pass

### CODE REVIEW

File: paddle2onnx/op_mapper/nn.py [hunk 1]
```diff
@@ -50,9 +50,15 @@ def opset_1(cls, graph, node, **kw):
             attrs['auto_pad'] = 'VALID'
         else:
             attrs['pads'] = pads
+        input_name = graph.get_name(node.input('Input', 0), with_remove=True)
+        # graph.make_node(
```
Comment (jiangjiajun) [inline paddle2onnx/op_mapper/nn.py]: remove useless codes

File: tests/onnxbase.py [hunk 2]
```diff
@@ -29,6 +29,24 @@ def compare(result, expect, delta=1e-10, rtol=1e-10):
     """
     if type(result) == np.ndarray:
         expect = np.array(expect)
+        # logging.info(result - expect)
```
Comment (jiangjiajun) [inline tests/onnxbase.py]: looks like some codes for debug

File: tests/onnxbase.py [hunk 3]
```diff
@@ -62,9 +80,9 @@ class APIOnnx(object):
     """
 
     def __init__(self, func, name, ver_list, delta=1e-6, rtol=1e-5):
-        self.seed = 33
-        np.random.seed(self.seed)
-        paddle.seed(self.seed)
+        # self.seed = 33
```
Comment (jiangjiajun) [inline tests/onnxbase.py]: +1

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

**Comment by jiangjiajun** [comment]
@yeliang2258 Let's put a simple technical document here to describe how this works. I noticed there are some new variables and functions(name_dict, change_dict, update_graph.....),  I think it's better that we add some annotations to make the pull request more reviewable.

File: paddle2onnx/op_mapper/nn.py (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/op_mapper/nn.py]: done

File: tests/onnxbase.py (hunk 3 above)
Comment (yeliang2258) [reply tests/onnxbase.py]: removed, done

File: tests/onnxbase.py (hunk 2 above)
Comment (yeliang2258) [reply tests/onnxbase.py]: removed, done

File: paddle2onnx/op_mapper/nn.py (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/op_mapper/nn.py]: > @yeliang2258 Let's put a simple technical document here to describe how this works. I noticed there are some new variables and functions(name_dict, change_dict, update_graph.....), I think it's better that we add some annotations to make the pull request more reviewable.

Yes, I think this is necessary, and I will submit relevant technical documents later.

Comment (yeliang2258) [reply paddle2onnx/op_mapper/nn.py]: > @yeliang2258 Let's put a simple technical document here to describe how this works. I noticed there are some new variables and functions(name_dict, change_dict, update_graph.....), I think it's better that we add some annotations to make the pull request more reviewable.

Yes, I think this is necessary, and I will submit relevant documents later.

**Comment by jiangjiajun** [comment]
- 代码中paddle处理数据的逻辑需改为numpy
- 变量的命名须遵守pep8标准
- 对于resnet结构的处理，改为pass后处理，简化逻辑

**Comment by yeliang2258** [comment]
related issue: https://github.com/PaddlePaddle/PaddleOCR/issues/5754

**Pull request closed without merging**